augur/data/schema-multiplot-collections.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://nextstrain.org/schemas/multiplot/v1",
+    "title": "Nextstrain multiplot data schema proposal with collection at the top level",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["collections"],
+    "properties": {
+        "default_collection": {
+            "description": "Default collection to display where the value must be the `key` in one of the objects of the collections array. Optional -- if not provided, first collection will be displayed",
+            "type": "string"
+        },
+        "collections": {
+            "description": "Collections of measurements and their configurations for display in auspice. Order of the collections determines the order they are shown in the collections dropdown.",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "description": "A single collection of related measurements and the collection's display config",
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["key", "groupings", "x_axis_label", "measurements"],
+                "properties": {
+                    "key": {
+                        "description": "The short name of the collection that is only used internally within auspice. Each collection is expected to have a unique key.",
+                        "type": "string"
+                    },
+                    "title": {
+                        "description": "The title to display in the collections dropdown and panel title. Optional -- if not provided, then `key` will be used",
+                        "type": "string"
+                    },
+                    "fields": {
```

Comment:
Any reason for the change from "groupings" to "fields"? I find the former more descriptive.